src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Card, Space, Typography, Button } from 'antd';
+
+import { useAuth } from 'services/AuthProvider';
+import './Dashboard.scss';
+import { useHistory } from 'react-router-dom';
+
+const { Text } = Typography;
+
+function Dashboard() {
+  const { currentUser, logout } = useAuth();
+  const history = useHistory();
+
+  async function handleLogout() {
+    try {
+      await logout();
+      history.push('/login');
```

Comment:
use static routes instead of putting in the string. Ideally this should look like

```js
import {routes} from 'routes'

history.push(routes.LOGIN)
```